src/printapp/static/js/app.js

```diff
@@ -140,7 +140,38 @@ App.SignInController = Ember.Controller.extend({
 });
 
 App.QueueController = Ember.ArrayController.extend({
-  itemController: 'queueItem'
+  needs: ['application','queue'],
+  itemController: 'queueItem',
+
+  actions : {
+
+    deleteJob: function(job_id) {
+      $("#" + job_id + " .loading-spinner").css("display", "inline-block");
+      $("#" + job_id + " a").hide();
+      var controller = this;
+
+      $.post('/api/deletejob/' + job_id)
+      .done(function(data, textStatus, response) {
+        var queue = controller.get('model');
+        var index = 0;
+        for (var i = 0; i < queue.length; i++) {
```

Comment:
Actually, use ember arraycontroller's `find` method instead of underscore.

http://emberjs.com/api/classes/Ember.ArrayController.html#method_find